ignf_gpf_api/store/Annexes.py

```diff
@@ -0,0 +1,70 @@
+from typing import List, Optional
+import requests
+
+from ignf_gpf_api.io.ApiRequester import ApiRequester
+from ignf_gpf_api.store.StoreEntity import StoreEntity
+from ignf_gpf_api.store.interface.PartialEditInterface import PartialEditInterface
+from ignf_gpf_api.store.interface.ReUploadFileInterface import ReUploadFileInterface
+from ignf_gpf_api.store.interface.DownloadInterface import DownloadInterface
+from ignf_gpf_api.store.interface.CreatedByUploadFileInterface import CreatedByUploadFileInterface
+
+
+class Annexes(CreatedByUploadFileInterface, DownloadInterface, PartialEditInterface, ReUploadFileInterface, StoreEntity):
+    """Classe Python représentant l'entité Fichier statique (annexes).
+
+    Cette classe permet d'effectuer les actions spécifiques liées aux fichiers statiques : création,
+    remplacement, mise à jour, suppression.
+    """
+
+    _entity_name = "annexes"
+    _entity_title = "annexes"
```

Comment:
Pour que ce soit uniforme, ne pas mettre au pluriel, il faudrait également renommer le fichier...
```suggestion
class Annexe(CreatedByUploadFileInterface, DownloadInterface, PartialEditInterface, ReUploadFileInterface, StoreEntity):
    """Classe Python représentant l'entité Annexe (annexe).

    Cette classe permet d'effectuer les actions spécifiques liées aux annexe : création,
    remplacement, mise à jour, suppression.
    """

    _entity_name = "annexe"
    _entity_title = "annexe"
```